src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -6781,9 +6879,6 @@ extends A
 ";
             var comp = CreateCompilationWithIL(new[] { source, IsExternalInitTypeDefinition }, ilSource: ilSource, parseOptions: TestOptions.Regular9);
             comp.VerifyEmitDiagnostics(
-                // (2,19): error CS8864: Records may only inherit from object or another record
-                // public record C : B
-                Diagnostic(ErrorCode.ERR_BadRecordBase, "B").WithLocation(2, 19),
                 // (4,29): error CS8871: 'C.PrintMembers(StringBuilder)' does not override expected method from 'B'.
                 //     protected override bool PrintMembers(System.Text.StringBuilder builder) => throw null;
                 Diagnostic(ErrorCode.ERR_DoesNotOverrideBaseMethod, "PrintMembers").WithArguments("C.PrintMembers(System.Text.StringBuilder)", "B").WithLocation(4, 29)
```

Comment:
>ERR_DoesNotOverrideBaseMethod [](start = 37, length = 29)

I would expect the same error reported for a record like this:
```
public record C : B;
```

Do we have a test for a scenario like this?